server/src/storage/customers.ts

```diff
@@ -14,11 +14,27 @@
  * limitations under the License.
  */
 
-import {CUSTOMER_KIND} from '../constants/kinds';
-import {CustomerResponse} from '../interfaces';
-import {getWithId} from './datastore';
+import {CUSTOMER_KIND, COMMIT_KIND} from '../constants/kinds';
+import {CustomerResponse, CommitResponse} from '../interfaces';
+import {getWithId, getAllWithId} from './datastore';
 
-const getCustomer = async (customerId: string): Promise<CustomerResponse> =>
-  getWithId(CUSTOMER_KIND, customerId);
+const getCustomer = async (customerId: number): Promise<CustomerResponse> => {
+  const customer = await getWithId(CUSTOMER_KIND, customerId);
+  const commits: CommitResponse[] = await getAllWithId(COMMIT_KIND, [
+    {
+      property: 'customerId',
+      value: customerId,
+    },
+    {
+      property: 'commitStatus',
+      value: 'ongoing',
+    },
+  ]);
+
+  return {
+    ...customer,
+    numCommits: commits.length,
```

Comment:
Is there a count api that we can call on the datastore?

Reply:
I can't find any for node.js!